d3a_api_client/aggregator.py

```diff
@@ -152,11 +155,25 @@ def execute_batch_commands(self):
             'batch-commands', {"aggregator_uuid": self.aggregator_uuid, "batch_commands": batch_command_dict})
         if posted:
             self._client_command_buffer.clear()
-            return self.dispatcher.wait_for_command_response('batch_commands', transaction_id)
+            response = self.dispatcher.wait_for_command_response('batch_commands', transaction_id)
+            for asset_uuid, responses in response["responses"].items():
+                for command_response in responses:
+                    log_bid_offer_confirmation(command_response)
+                    log_deleted_bid_offer_confirmation(
+                        command_response,
+                        asset_name=get_name_from_area_name_uuid_mapping(self.area_name_uuid_mapping,
+                                                                        asset_uuid))
+            return response
 
     def get_uuid_from_area_name(self, name):
         return get_uuid_from_area_name_in_tree_dict(self.area_name_uuid_mapping, name)
 
+    def get_name_from_area_uuid(self, asset_uuid):
+        for area_name, area_uuids in self.area_name_uuid_mapping.items():
+            for area_uuid in area_uuids:
+                if area_uuid == asset_uuid:
+                    return area_name
```

Comment:
My old comment https://github.com/gridsingularity/d3a-api-client/pull/140#discussion_r620198602 referred to  this part too.

Reply:
@faizan2590 done